src/lint/config/index.ts

```diff
@@ -0,0 +1,25 @@
+import ConfigSchema, { Config, RulesPreset } from "./schema";
+import presets, { defaultPreset, DEFAULT_PRESET_NAME } from "../presets";
+
+export const CONFIG_FILE_NAME = ".gitlintrc";
```

Comment:
Same here, or alternatively, do you want to use CAPITAL_SNAKE_CASE for all constants? If so, please also change other places (e.g. `msInDay`).